target_postgres/tests/test_standard_target.py

```diff
@@ -286,7 +286,9 @@ def test_relational_data(postgres_target):
         ]
 
         full_table_name = f"{schema_name}.test_users"
-        result = connection.execute(f"SELECT * FROM {full_table_name} ORDER BY id")
+        result = connection.execute(
+            sqlalchemy.text(f"SELECT * FROM {full_table_name} ORDER BY id")
```

Comment:
there's a lot of these where I just had to wrap the query string with sqlalchemy.text(). connection.execute() no longer accepts a string as an argument and sqlalchemy.text() constructs the required Executable.